test/test_serde.py

```diff
@@ -333,21 +333,6 @@ def test_compressed_serde(compress_scheme):
     assert numpy.array_equal(arr, arr_serialized_deserialized)
 
 
-@pytest.mark.parametrize("compress_scheme", [1, 2, 3, 100])
-def test_invalid_decompression_scheme(compress_scheme):
```

Comment:
why is this test removed

Reply:
The test was not valid actually: you can't overwrite `serde._apply_compress_scheme = some_other_compression_scheme` like it is done here, it doesn't work